src/DatabaseConnection.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ProjectManagementApi;
+
+use PDO;
+
+class DatabaseConnection extends PDO
+{
+    public function __construct(string $dsn, string $username = null, string $password = null, array $options = null)
```

Comment:
just to be 100% sure we're not changing the original signature of the method, it would be best to add questionmarks on the type hints like in the PDO constructor:
![image](https://user-images.githubusercontent.com/25844355/227171732-453d2c8a-1580-4bfe-a485-a669e2f54051.png)

Reply:
done